tutorial/pytorch/1_tensors.ipynb

```diff
@@ -291,12 +291,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.8"
+   "version": "3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)]"
   },
   "orig_nbformat": 4,
   "vscode": {
    "interpreter": {
-    "hash": "aee646b5f0c348debdea9b36e04014d3b501ba8b83876c774c538990ffd76616"
+    "hash": "d0b1cd08f25460b8eca4806833a13a3bdd20cef72802d0ad364ee4ea93f06117"
```

Comment:
Commiting such changes makes your PR messy.

Reply:
I will watch out for these changes in the commit process and will drop the changes in the future,